test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs

```diff
@@ -144,25 +136,24 @@ void ProcessExport(Batch<Metric> batch)
             var counterLong = meter1.CreateCounter<long>("name1");
             counterLong.Add(10);
             metricReader.Collect();
-            Assert.Equal(1, metricCount);
+            Assert.Single(metricItems);
 
             // The following will be ignored as
             // metric of same name exists.
             // Metric stream will remain one.
+            // TODO: Fix comments and Assert statements to actually ensure only 1 metric stream is present.
             var anotherCounterSameName = meter1.CreateCounter<long>("name1");
             anotherCounterSameName.Add(10);
-            metricCount = 0;
```

Comment:
How would I replicate the `metricCount = 0;` statement when using the `InMemoryExporter`?
I modified the test in a way so that it would pass, but I believe all the `Assert` statements should be `Assert.Single(metricItems)` rather than the count increasing from 1 to 2 to 3.

Reply:
we can use metricItems.Count, right?

Reply:
metricItems.Clear() should be done, in lieu of metriCount=0;